manual/src/refman/extensions/arrayliterals.etex

```diff
@@ -0,0 +1,74 @@
+(Introduced in OCaml 5.4)
```

Comment:
You are missing the forward links in `expr.etex` and `patterns.etex`.

Reply:
Fixed, thanks.